guides/common/modules/proc_configuring-smart-proxy-to-provision-secure-boot-enabled-hosts.adoc

```diff
@@ -0,0 +1,113 @@
+[id="configuring-{smart-proxy-context}-to-provision-{client-os-context}-on-Secure-Boot-enabled-hosts"]
+= Configuring {SmartProxy} to provision {client-os} on Secure Boot enabled hosts
+
+Secure Boot follows a chain of trust from the start of the host to the loading of Linux kernel modules.
+The first shim that is loaded determines which distribution can be booted or loaded by using a `kexec` system call until the next reboot.
+
+To provision {client-os} on Secure Boot enabled hosts with the *Grub2 UEFI SecureBoot* and *Grub2 UEFI HTTPS SecureBoot* PXE loaders, you have to provide signed shim and GRUB2 binaries provided by the vendor of your operating system.
+
+[IMPORTANT]
+====
+ifdef::satellite[]
+You have to perform the following configuration steps on each TFTP {SmartProxy} for a subnet to provision Secure Boot enabled hosts on that subnet.
+endif::[]
+ifndef::satellite[]
+You have to perform the following configuration steps on each TFTP proxy for a subnet to provision Secure Boot enabled hosts on that subnet.
+endif::[]
+====
+
+The following example works for {client-os} on x86_64 architecture.
+
+.Prerequisites
+ifeval::["{client-os}" == "Debian"]
+* Ensure that `ar` and `xz` are installed on your {SmartProxy}.
+endif::[]
+ifeval::["{client-os}" == "Ubuntu"]
+* Ensure that `ar`, `xz`, and `zstd` are installed on your {SmartProxy}.
+endif::[]
+ifeval::["{client-pkg-ext}" == "rpm"]
+* Ensure that `cpio` is installed on your {SmartProxy}.
+endif::[]
+
+.Procedure
```

Comment:
If this is all common, why don't we ship a helper script that takes several inputs and saves it to the correct places? That makes both the procedure and testing easier. We can also ensure all commands we use are properly ensured via packaging dependencies.

There's precedent for this: https://github.com/theforeman/smart-proxy/blob/develop/sbin/foreman-prepare-realm is installed ([here](https://github.com/theforeman/foreman-packaging/blob/067c3b95706b2ebf112aa4cf096b39caab6778b4/packages/foreman/foreman-proxy/foreman-proxy.spec#L135) for RPM). Then the documentation refers to this:
https://github.com/theforeman/foreman-documentation/blob/3259af88086c536d683e3d2846172ca5051d5b4d/guides/common/modules/con_configuring-project-to-manage-the-lifecycle-of-a-host-registered-to-a-freeipa-realm.adoc#L29-L33